web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/AutoDeploy.tsx

```diff
@@ -247,49 +257,84 @@ const CreateAccessRole = ({
   taskRoleArn,
   setTaskRoleArn,
   disabled,
+  dbMeta,
+  validator,
 }: {
   taskRoleArn: string;
   setTaskRoleArn(r: string): void;
   disabled: boolean;
+  dbMeta: DbMeta;
+  validator: Validator;
 }) => {
+  const [scriptUrl, setScriptUrl] = useState('');
+  const { integration, selectedAwsRdsDb } = dbMeta;
+
+  function generateAutoConfigScript() {
+    if (!validator.validate()) {
+      return;
+    }
+
+    const newScriptUrl = cfg.getDeployServiceIamConfigureScriptUrl({
+      integrationName: integration.name,
+      region: selectedAwsRdsDb.region,
+      // arn's are formatted as `don-care-about-this-part/role-arn`.
+      // We are splitting by slash and getting the last element.
+      awsOidcRoleArn: integration.spec.roleArn.split('/').pop(),
+      taskRoleArn,
+    });
+
+    setScriptUrl(newScriptUrl);
+  }
+
   return (
     <StyledBox mb={5}>
       <Text bold>Step 1</Text>
-      <Text mb={2}>Create an Access Role for the Database Service</Text>
+      <Text mb={2}>
+        Name a Task Role ARN for this Database Service and generate a configure
+        command. This command will configure the required permissions in your
+        AWS account.
+      </Text>
       <FieldInput
         mb={4}
         disabled={disabled}
-        rule={requiredField('Task Role ARN is required')}
+        rule={roleArnMatcher}
         label="Name a Task Role ARN"
         autoFocus
         value={taskRoleArn}
-        placeholder="teleport"
-        width="400px"
+        placeholder="TeleportDatabaseAccess"
+        width="440px"
         mr="3"
         onChange={e => setTaskRoleArn(e.target.value)}
-        toolTipContent="Lorem ipsume dolores"
+        toolTipContent={`Amazon Resource Names (ARNs) uniquely identifies AWS \
+        resources, in this case you will be naming an IAM role that this \
+        deployed service will be using`}
```

Comment:
Not sure if this is exactly correct

```suggestion
        toolTipContent={`Amazon Resource Names (ARNs) uniquely identify AWS \
        resources. In this case you will be giving an ARN to the IAM role that this \
        deployed service will be using`}
```

If not then

```suggestion
        toolTipContent={`Amazon Resource Names (ARNs) uniquely identify AWS \
        resources. In this case you will naming an IAM role that this \
        deployed service will be using`}
```